src/schemas/Clinical/index.ts

```diff
@@ -593,10 +597,13 @@ const resolvers = {
       context: GlobalGqlContext,
         ) => {
           const { Authorization } = context;
-
-          const donorEntity = parent.clinicalEntities.find(({entityName}) => entityName === CoreClinicalEntities.donor);
-          const donorIds = donorEntity?.completionStats && donorEntity?.completionStats.map(donor => donor.donorId);
-          
+          const entity = parent.clinicalEntities[0];
```

Comment:
why the switch from matching names (line 597 in the original) to getting the first array item? I assume we only have one item in `clinicalEntities` but will that always be the case...?

Reply:
That is a very good catch, I've updated this logic to do a forEach loop over parent.clinicalEntities and find DonorIds for all Entities

The side menus / side-bars need to find which entities have errors so it has to query multiple entities
The submitted data page only displays 1 entity at a time on the page so that's why I was using `clinicalEntities[0]`

Reply:
nice fix, so this'll work for 1 entity on Submitted Data & N entities in the side menu?